tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/FunExceptRule.scala

```diff
@@ -161,6 +163,19 @@ class FunExceptRule(rewriter: SymbStateRewriter) extends RewritingRule {
     rewriter.rewriteUntilDone(nextState.setRex(newRecord.toNameEx))
   }
 
+  def rewriteRowRec(
+      state: SymbState,
+      oldRecord: ArenaCell,
+      indexEx: TlaEx,
+      newValue: ArenaCell): SymbState = {
+    val fieldToUpdate = indexEx match {
+      case ValEx(TlaStr(key)) => key
+      case ex => throw new RewriterException("Expected a string when updating a record, found: " + ex, ex)
```

Comment:
Seems like the error for this case and the error for `getFieldName` should have more or less the same language and format?